doc/WEATHER_TYPE.md

```diff
@@ -20,46 +19,27 @@ Each weather type is a type of weather that occurs, its effects and what causes
-    snowstorm and snow. |
+| `sound_category`               | Optional, if playing sound effects what to use. Valid values are: silent, drizzle, rainy, thunder, flurries, snowstorm and snow. |
 | `sun_intensity`                | Strength of the sun. Valid values are: none, light, normal, and high  |
 | `duration_min`                 | Optional, the lower bound on the amount of time this weather can last. Defaults to 5 minutes. Unless time_between_min and time_between_max are set the weather can happen again as soon as it ends. |
 | `duration_max`                 | Optional, the upper bound on the amount of time this weather can last. Defaults to 5 minutes. Unless time_between_min and time_between_max are set the weather can happen again soon as it ends. |
 | `time_between_min`             | Optional, the lower bounds of the amount of time that will be guranteed to pass before this weather happens again. Defaults to 0. |
 | `time_between_max`             | Optional, the upper bounds of the amount of time that will be guranteed to pass before this weather happens again. Defaults to 0. |
 | `weather_animation`            | Optional, Information controlling weather animations.  Members: factor, color and glyph |
 | `effects`                      | Array for the effects the weather has. Descibed in detail below
-| `requirements`                 | Optional, is what determines what weather it is.  All members are optional. 
-	When checking what weather it is it loops through the entries in order and uses the last one to succeed. |
-	
-	`pressure_min`
-	`pressure_max`
-	`humidity_min`
-	`humidity_max`
-	`temperature_min`
-	`temperature_max`
-	`windpower_min`
-	`windpower_max`
-	| These are all minimum and maximum values for which the weather will occur.  I.e it will only rain if its humid enough |
-	
-	`humidity_and_pressure`      | if there are pressure and humidity requirements are they both required or just one |
-	`acidic`                     | does this require acidic precipitation                                |
-	`time`                       | Valid values are: day, night, and both.                               |
-	`required_weathers`          | a string array of possible weathers it is at this point in the loop. i.e. rain can only happen if the conditions for clouds light drizzle or drizzle are present |
-	`time_passed_min`            | Optional: Until this much time post cataclysm this weather won't happen.|
-	`time_passed_max`            | Optional: After this much time post cataclysm this weather stops.|
-	`one_in_chance`              | Optional: This has a 1 in this value chance of happening.  This will usually be called every 5 minutes|
-	
-### Example
+| `requirements`                 | Optional, is what determines what weather it is.  All members are optional. When checking what weather it is it loops through the entries in order and uses the last one to succeed. |
```

Comment:
How about something like

> Optional, a list of conditions that have to be met for the weather to appear.

Reply:
And does "succeed" include just on the requirements, or on the chance of a change to that weather  type from the current weather?